Userland/Utilities/dd.cpp

```diff
@@ -40,11 +40,11 @@ static String split_at_equals(const char* argument)
     String string_value(argument);
 
     auto values = string_value.split('=');
```

Comment:
You can use `string_value.split_limit('=', 1);` to get at most one split :)

Reply:
Thanks! This looks much simpler and comprehensible. I have made the changes.